.env.example

```diff
@@ -1,25 +1,26 @@
 # Example .env file for selfhosting the bot
 # Don't add your .env file to your github repository, these settings are used to access the bot
 
-
+# To edit/set the settings, remove this line and the one below it
+READONLY=true
 # Your bot token to login your bot
-token="<Discord bot token>"
+token=""
```

Comment:
The variables should be filled as this is an example file showing how the variables should be filled.